core/src/main/java/org/elasticsearch/action/ingest/IngestActionFilter.java

```diff
@@ -54,9 +54,9 @@ public IngestActionFilter(Settings settings, IngestBootstrapper bootstrapper) {
 
     @Override
     public void apply(Task task, String action, ActionRequest request, ActionListener listener, ActionFilterChain chain) {
-        String pipelineId = request.getFromContext(IngestPlugin.PIPELINE_ID_PARAM_CONTEXT_KEY);
+        String pipelineId = request.getFromContext(ConfigurationUtils.PIPELINE_ID_PARAM_CONTEXT_KEY);
```

Comment:
maybe just move this constant to this class? It is only used here and the test testing this class.